cmd/todo/main.go

```diff
@@ -14,13 +14,15 @@ import (
 
 const (
 	todoFile = "data/todos.mayur"
+	historyFile = "data/history.mayur"
```

Comment:
We have to add the completed tasks to the history file.

Reply:
Working on it.